assimilation_code/modules/assimilation/filter_mod.dopplerfold.f90

```diff
@@ -392,30 +402,54 @@ subroutine filter_main()
 
 call trace_message('Before initializing inflation')
 
-call validate_inflate_options(inf_flavor, inf_damping, inf_initial_from_restart, &
+inflation_flavor = set_inflation_flavor(inf_flavor)
```

Comment:
why is a separate variable needed now?  maybe this needs a comment to explain why using inf_flavor isn't sufficient anymore

Reply:
I cannot change the namelist variable name ... so 'inf_flavor' must now be a character instead of integer variable.
I needed an integer variable everywhere the old 'inf_flavor' was used, so I had to make a new variable.

Reply:
perfect explanation.  so:

```suggestion
! inf_flavor from the namelist is now a character string.  
! the inflation-related subroutines require an integer so the
! variable inflation_flavor is added and is type integer.
inflation_flavor = set_inflation_flavor(inf_flavor)
```

Reply:
Since I had to add it to both filter_mod.f90 and filter_mod.f90 ... I did this on the command line and pushed.